src/ex1/script.js

```diff
@@ -1,50 +1,111 @@
 const input = document.querySelector(".todo-input");
```

Comment:
there is some logic you added here that could be in a new class like you did really nice with PokemonClient or ItemManager.